src/SlateAsInputEditor/index.js

```diff
@@ -428,7 +428,7 @@ const SlateAsInputEditor = React.forwardRef((props, ref) => {
 
   return (
     <div>
-      <ToolbarWrapper {...editorProps} id="slate-toolbar-wrapper-id" />
+      { props.readOnly ? '' : <ToolbarWrapper {...editorProps} id="slate-toolbar-wrapper-id" /> }
```

Comment:
Could this be changed to `{ !props.readOnly && <ToolbarWrapper ...`?

Reply:
@irmerk It's done, please have a look